dataprofiler/profilers/unstructured_labeler_profile.py

```diff
@@ -47,15 +48,91 @@ def __init__(self, data_labeler_dirpath=None, options=None):
         self.separators = (' ', ',', ';', '"', ':', '\n', '\t', ".", "!", "'")
         self.times = defaultdict(float)
 
+    def __add__(self, other):
+        """
+        Merges the properties of two UnstructuredLabelerProfile
+
+        :param self: first profile
+        :param other: second profile
+        :type self: UnstructuredLabelerProfile
+        :type other: UnstructuredLabelerProfile
+        :return: New UnstructuredLabelerProfile merged profile
+        """
+        
+        if not isinstance(other, UnstructuredLabelerProfile):
+            raise TypeError("Unsupported operand type(s) for +: "
+                            "'UnstructuredLabelerProfile' and '{}'".format(
+                other.__class__.__name__))
+        
+        if self.data_labeler != other.data_labeler:
+            raise AttributeError("Cannot merge. The data labeler is not the "
+                                 "same for both profiles.")
+
+        # recreate options so the DataLabeler is transferred and not duplicated
+        options = DataLabelerOptions()
+        options.data_labeler_object = self.data_labeler
+
+        merged_profile = UnstructuredLabelerProfile(options=options)
+        merged_profile.entity_counts = merged_profile.\
+            add_nested_dictionaries(self.entity_counts, other.entity_counts)
+
+        merged_profile.char_sample_size = self.char_sample_size + \
+                                          other.char_sample_size
+        merged_profile.word_sample_size = self.word_sample_size + \
+                                          other.word_sample_size
+        
+        merged_profile.times = merged_profile.\
+            add_nested_dictionaries(self.times, other.times)
+        
+        merged_profile._update_percentages()
+
+        return merged_profile
+
     @property
     def label_encoding(self):
         return self.data_labeler.labels
 
+
+    def add_nested_dictionaries(self, first_dict, second_dict):
```

Comment:
Since many profilers could use this we should consider putting it in a base class or utility func

Reply:
Also, I wonder if a cleaner solution would be to copy the first dict and then merge in the values from the second as oppose to looping over both. Not saying it would be, but just a thought